package.json

```diff
@@ -262,7 +262,7 @@
     "tar-utils": "^2.1.0",
     "through2": "^2.0.3",
     "tmp": "^0.2.1",
-    "typed-error": "^3.2.0",
+    "typed-error": "^3.2.1",
```

Comment:
Updates its dev deps, just doing the safe update to reduce the noise from `npm outdated`